sarpy/geometry/geocoords.py

```diff
@@ -175,38 +141,57 @@ def wgs_84_norm(x, y=None, z=None):
 
 
 def ric_ecf_mat(rarp, varp, frame_type):
-    """Compute ECF transformation matrix for RIC frame"""
+    """
+    Computes the ECF transformation matrix for RIC frame.
 
-    if frame_type == 'eci':  # RIC_ECI frame
+    :param rarp:
+    :param varp:
+    :param frame_type: one of 'eci' or 'ecf'?
+    :return:
+    """
+    # TODO: LOW - fix this docstring. These argument names are poor.
+
+    if frame_type.upper() == 'ECI':
         vi = varp + np.cross([0, 0, w], rarp)
-    elif frame_type == 'ecf':  # RIC_ECF frame
+    elif frame_type.upper() == 'ECF':
         vi = varp
+    else:
+        ValueError('frame_type must be one of "ECI", "ECF"')
 
-    r = rarp/np.sqrt(np.sum(np.power(rarp, 2)))
+    r = rarp/np.sqrt(np.sum(rarp*rarp))  # what if it's zero?
```

Comment:
In addition to being tidier, I'm one hundred percent confident that item*item is uniformly better than np.power(item, 2). It think speed advantage of arithmetic usually starts breaking down at 4th power. 

That being said, this was to merely tidy up the appearance, and I don't actually feel passionately because both are quite fast.